src/components/TablePreview/TablePreview.tsx

```diff
@@ -51,17 +52,23 @@ const TablePreview: FC<Props> = ({}) => {
                                     </td>
                                     <td style={name.style}>{name.textValue}</td>
                                     <td style={spacerStyle}></td>
-                                    <td style={phone.style}>{phone.textValue}</td>
+                                    <td style={phone.style}>
+                                        <a href="tel:${phone.textValue}"> {phone.textValue}</a>
+                                    </td>
                                 </tr>
                                 <tr>
                                     <td style={title.style}>{title.textValue}</td>
                                     <td style={spacerStyle}></td>
-                                    <td style={email.style}>{email.textValue} </td>
+                                    <td style={email.style}>
+                                        <a href="mailto:${email.textValue}">{email.textValue}</a>
```

Comment:
Almost works! But when I try to click the hyperlink it points me to `${email.textValue}` 🙈 

<img width="920" alt="image" src="https://github.com/evargast/signature-generator/assets/52969178/9b09d51e-84e8-4acb-a92f-a80975098de3">

[Template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) might come in handy to fix this one